sycl/include/CL/sycl/INTEL/esimd/detail/esimd_memory_intrin.hpp

```diff
@@ -693,13 +723,15 @@ __esimd_media_block_load(unsigned modififer, TACC handle, unsigned plane,
 }
 
 template <typename Ty, int M, int N, typename TACC>
-inline void
-__esimd_media_block_store(unsigned modififer, TACC handle, unsigned plane,
-                          unsigned width, unsigned x, unsigned y,
-                          sycl::INTEL::gpu::vector_type_t<Ty, M * N> vals) {
-  unsigned bpp = sycl::INTEL::gpu::AccessorPrivateProxy::getElemSize(handle);
+inline void __esimd_media_block_store(unsigned modififer, TACC handle,
+                                      unsigned plane, unsigned width,
+                                      unsigned x, unsigned y,
+                                      __SIGD::vector_type_t<Ty, M * N> vals) {
+  unsigned bpp =
+      sycl::INTEL::gpu::detail::AccessorPrivateProxy::getElemSize(handle);
   unsigned vpp = bpp / sizeof(Ty);
-  auto range = sycl::INTEL::gpu::AccessorPrivateProxy::getImageRange(handle);
+  auto range =
+      sycl::INTEL::gpu::detail::AccessorPrivateProxy::getImageRange(handle);
```

Comment:
Minor comment: you could also use __SIGD here instead of sycl::INTEL::gpu::detail

Reply:
Right, updated